source/BoardingPanel.cpp

```diff
@@ -72,6 +72,10 @@ BoardingPanel::BoardingPanel(PlayerInfo &player, const shared_ptr<Ship> &victim)
 	for(const auto &it : victim->Outfits())
 		if(!it.first->Get("unplunderable"))
 			plunder.emplace_back(it.first, it.second);
+
+	// Outfits stored in cargo can always be plundered.
+	for(const auto &it : victim->Cargo().Outfits())
+		plunder.emplace_back(it.first, it.second);	
```

Comment:
This alone won't work correctly, because later on if you decide to plunder that outfit, the game will try to remove it from the ship itself rather than its cargo hold:

```C++
			// Transfer as many as possible of these outfits to your cargo hold.
			count = cargo.Add(outfit, count);
			victim->AddOutfit(outfit, -count);
```